xbmc/network/NetworkServices.cpp

```diff
@@ -829,12 +829,9 @@ bool CNetworkServices::StartUPnP()
   ret |= StartUPnPClient();
   if (CSettings::GetInstance().GetBool(CSettings::SETTING_SERVICES_UPNPSERVER))
   {
-   ret |= StartUPnPServer();
-  }
-
-  if (CSettings::GetInstance().GetBool(CSettings::SETTING_SERVICES_UPNPCONTROLLER))
-  {
-    ret |= StartUPnPController();
+    ret |= StartUPnPServer(); 
+    if (CSettings::GetInstance().GetBool(CSettings::SETTING_SERVICES_UPNPCONTROLLER))
+      ret |= StartUPnPController();
   }
 
   if (CSettings::GetInstance().GetBool(CSettings::SETTING_SERVICES_UPNPRENDERER))
```

Comment:
Could you add a second commit to remove the curly braces belonging to this `if` statement?